docs/guides/metrics/starter-kit/README.md

```diff
@@ -0,0 +1,107 @@
+<!-- Header block for Apache DevLake Starter Kit -->
```

Comment:
Recommend putting this `README.md` file at the top-level of your `/metrics` folder. This should be the starting point for the guide, and link to all your other relevant files / artifacts.